examples/video/outstream/pb-ve-outstream-no-server.html

```diff
@@ -25,105 +26,59 @@ <h1>{{ page.title }}</h1>
       </p>
     </div>
 
-    <div>
-      <div>
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-          aliquip ex ea commodo consequat. Duis aute irure dolor in
-          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-          culpa qui officia deserunt mollit anim id est laborum.
-        </p>
-      </div>
-
-      <div id="video1">
-        <p>Prebid Outstream Video Ad</p>
-      </div>
+{% capture htmlCodePrebid %}
+<script src="https://cdn.michao-ssp.com/original/prebid.js"></script>
```

Comment:
I would leave this PR open until https://github.com/prebid/Prebid.js/pull/12507 is merged and it can be replaced with "not-for-prod" distribution, so you don't have to maintain this distribution